api/rppdefs.h

```diff
@@ -468,6 +468,24 @@ typedef enum
     REFLECT
 } RpptAudioBorderType;
 
+/*! \brief RPPT Border Padding Vertical Direction
+ * \ingroup group_rppdefs
+ */
+typedef enum
+{
+    TOP_EDGE = 0,
+    BOTTOM_EDGE
+} RpptBorderVerticalDirection;
+
+/*! \brief RPPT Border Padding Horizontal Direction
+ * \ingroup group_rppdefs
+ */
+typedef enum
+{
+    LEFT_EDGE = 0,
+    RIGHT_EDGE
+} RpptBorderHorizontalDirection;
```

Comment:
Just like the one below, call this RpptImageBorderEdge instead of RpptImageBorderType.

Are two separate enums for horizontal and vertical needed, can one suffice?